include/envoy/stream_info/stream_info.h

```diff
@@ -357,6 +357,11 @@ class StreamInfo {
    */
   virtual bool hasAnyResponseFlag() const PURE;
 
+  /**
+   * @return response flags encoded as an integer.
+   */
+  virtual uint64_t getResponseFlags() const PURE;
```

Comment:
responseFlags for naming consistency (no get prefix)

Reply:
Done.